lib/tzinfo/ruby_core_support.rb

```diff
@@ -138,8 +138,11 @@ def self.open_file(file_name, mode, opts, &block)
         File.open(file_name, mode, &block)
       end
     else
-      def self.open_file(file_name, mode, opts, &block)
-        File.open(file_name, mode, opts, &block)
+      class << self
+        def open_file(file_name, mode, opts, &block)
+          ::File.open(file_name, mode, opts, &block)
+        end
+        ruby2_keywords :open_file if respond_to?(:ruby2_keywords, true)
```

Comment:
> /Users/kamipo/src/github.com/rails/rails/vendor/bundle/ruby/2.7.0/bundler/gems/tzinfo-71873adbf908/lib/tzinfo/ruby_core_support.rb:145: warning: Skipping set of ruby2_keywords flag for open_file (method accepts keywords or method does not accept argument splat)

We can only use `ruby2_keywords` when method accepts splat arguments.
In this case probably we need extra version check `RUBY_VERSION >= '2.0'` then `::File.open(file_name, mode, **opts, &block)`.